Dockerfile

```
@@ -0,0 +1,31 @@
+
+RUN apt update
+RUN apt install -y curl python3 python3-venv ffmpeg
+RUN python3 -m venv .venv
+ENV PATH="/home/ubuntu/.venv/bin:$PATH"
+RUN pip install -r requirements.txt
+
+COPY package.json ./
+COPY pnpm-workspace.yaml ./
+COPY pnpm-lock.yaml ./
+
+RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.0/install.sh | bash
```

Hadolint (2.12.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)